js/timex.js

```diff
@@ -176,6 +176,7 @@ module.exports = class timex extends Exchange {
                 },
             },
             'options': {
+                'expireIn': '31536000', // 365 × 24 × 60 × 60
```

Comment:
Do we have to define it here? I would keep it user-controllable, but not set it by default.